tests/components/netatmo/test_init.py

```diff
@@ -449,3 +451,59 @@ async def fake_ensure_valid_token(*args, **kwargs):
 
     for config_entry in hass.config_entries.async_entries("netatmo"):
         await hass.config_entries.async_remove(config_entry.entry_id)
+
+
+async def remove_device(
+    ws_client: MockHAClientWebSocket, device_id: str, config_entry_id: str
+) -> bool:
+    """Remove config entry from a device."""
+    await ws_client.send_json(
+        {
+            "id": 1,
+            "type": "config/device_registry/remove_config_entry",
+            "config_entry_id": config_entry_id,
+            "device_id": device_id,
+        }
+    )
+    response = await ws_client.receive_json()
+    return response["success"]
+
+
+async def test_device_remove_devices(
+    hass: HomeAssistant,
+    hass_ws_client: WebSocketGenerator,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+    config_entry,
+    netatmo_auth,
+) -> None:
+    """Test we can only remove a device that no longer exists."""
+
+    assert await async_setup_component(hass, "config", {})
+
+    with selected_platforms(["climate"]):
+        assert await hass.config_entries.async_setup(config_entry.entry_id)
+
+        await hass.async_block_till_done()
+
+    climate_entity_livingroom = "climate.livingroom"
+    entity = entity_registry.entities[climate_entity_livingroom]
```

Comment:
```suggestion
    entity = entity_registry..async_get(climate_entity_livingroom)
```
Not sure if this is the right function, but shouldn't we use something like this instead?

Reply:
That works equally well. Thanks.